File: BOJ/sloved.ac/새싹/배열/python/10807.py
"""
10807번
개수 세기

총 N개의 정수가 주어졌을 때, 정수 v가 몇 개인지 구하는 프로그램을 작성하시오.
https://www.acmicpc.net/problem/10807
"""
import sys

amount = int(sys.stdin.readline())
lists = list(map(int, sys.stdin.readline().rstrip().split()))
target = int(sys.stdin.readline())

result = 0

for i in lists:
    if i == target: result += 1

print(result)

File: BOJ/sloved.ac/새싹/배열/java/10807.java
/*
10807번
개수 세기

총 N개의 정수가 주어졌을 때, 정수 v가 몇 개인지 구하는 프로그램을 작성하시오.
https://www.acmicpc.net/problem/10807
*/
import java.io.*;
import java.util.*;

class Main{
  public static void main(String[] args) throws IOException{
    BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

    BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));

    int amount = Integer.parseInt(br.readLine());
    int lists[] = Arrays
      .stream(br.readLine().split(" "))
      .mapToInt(Integer::parseInt)
      .toArray();
    int target = Integer.parseInt(br.readLine());

    int sum = 0;

    for(int i : lists){
      if(i == target) sum++;
    }

    bw.write(String.valueOf(sum));
    bw.flush();
  }
}